container.go

```diff
@@ -262,18 +262,32 @@ func (c *Container) delete() error {
 	return nil
 }
 
-func (c *Container) start() error {
+// fetchRunningState retrieves the container state.
+//
+// cmd specifies the operation (or verb) that the retieval is destined
+// for and is only used to make the returned error as descriptive as
+// possible.
+func (c *Container) fetchRunningState(cmd string) (State, error) {
```

Comment:
I think you could simplify this function name (e.g fetchState), or use fetchStateRunningPod(), because I think the name can be a bit confusing as we can think that we are checking the container's state being "running". What do you think ?